spec/models/sales_log_spec.rb

```diff
@@ -80,21 +80,20 @@
     let(:organisation_3) { create(:organisation) }
 
     before do
-      create(:sales_log, :in_progress, owning_organisation: organisation_1, managing_organisation: organisation_1)
-      create(:sales_log, :completed, owning_organisation: organisation_1, managing_organisation: organisation_2)
-      create(:sales_log, :completed, owning_organisation: organisation_2, managing_organisation: organisation_1)
-      create(:sales_log, :completed, owning_organisation: organisation_2, managing_organisation: organisation_2)
+      create(:sales_log, :in_progress, owning_organisation: organisation_1)
+      create(:sales_log, :completed, owning_organisation: organisation_1)
+      create(:sales_log, :completed, owning_organisation: organisation_2)
     end
 
     it "filters by given organisation id" do
```

Comment:
Is it necessary to test that it works with both the org and the org id? AFAICT the fact that it works with both is due to some Rails magic rather than because of how the the scope has been written, so seems odd to test that.

Reply:
good shout, will remove